examples/tpm.yml

```diff
@@ -0,0 +1,30 @@
+kernel:
+  image: linuxkit/kernel:4.9.38
+  cmdline: "console=tty0 console=ttyS0"
+init:
+  - linuxkit/init:d049e7b2074da5cd699a27defb47eb101142455d
+  - linuxkit/runc:d5cbeb95bdafedb82ad2cf11cff1a5da7fcae630
+  - linuxkit/containerd:e33e0534d6fca88e1eb86897a1ea410b4a5d722e
+  - linuxkit/ca-certificates:67acf038c44bb191ebb704ec7bb39a1524052cdf
+onboot:
+  - name: sysctl
+    image: linuxkit/sysctl:d1a43c7c91e92374766f962dc8534cf9508756b0
+  - name: dhcpcd
+    image: linuxkit/dhcpcd:4b7b8bb024cebb1bbb9c8026d44d7cbc8e202c41
+    command: ["/sbin/dhcpcd", "--nobackground", "-f", "/dhcpcd.conf", "-1"]
+services:
+  - name: getty
+    image: linuxkit/getty:1b651a91f1c17f50357be9873b580ccf81668630
+    env:
+     - INSECURE=true
+  - name: tss
+    image: linuxkit/tss:3406973699199b47224ad4e819e5728964673772
+  - name: rngd
```

Comment:
We could have it up and then run, e.g. `tpm_version` or `tpm_nvinfo`, but they will fail if `tcsd` isn't running, and it will fail unless you test with a chip, or a virtual tpm. `tcsd` only supports real tpms, so you either need the one that loads into the kernel, or one that virtualization supports.

Happy to take some good ideas here.